### DISCUSSION TITLE
Plugin integration for static commands

### DISCUSSION THREAD

**Discussion by haxfn** [helix-editor/helix#12065] [Q&A] [answered]
Things like `normal_mode`, `extend_line`, `move_char_left` also termed as static commands (#12050).

Is there a plan to integrate then with `steel`? If so, what will be the interface?

**Comment by david-crespo** [comment, marked as answer]
Yes.

> The basic built in functions are first all of the function that are typed and static

https://github.com/helix-editor/helix/pull/8675

You can also see references to `static-commands` in recent comments.
